File: modul 1/c/PRAK110-2410817110008-Muhammad Azma Al Faqih.c
#include <stdio.h>
#include <math.h>

float alas = 5, tinggi = 12;
float miring;

int main(){
  miring =  sqrt(pow(alas,2)+pow(tinggi,2));
  printf("Diketahui :\n");
  printf("Alas = %.0f\n", alas);
  printf("Tinggi = %.0f\n\n", tinggi);
  printf("Jawab :\n");
  printf("Sisi A = %.f cm\n", tinggi);
  printf("Sisi B = %.f cm\n", miring);
  printf("Sisi C = %.f cm\n", alas);
  printf("Keliling = %.f cm\n", alas+tinggi+miring);
  printf("Luas = %.f cm", alas*tinggi/2);
}

File: modul 1/python/PRAK110-2410817110008-Muhammad Azma Al Faqih.py
import math

alas, tinggi = 5, 12
miring = math.sqrt(math.pow(alas, 2) + math.pow(tinggi,2))

print(f"Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")
print(f"Jawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {miring:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {alas+tinggi+miring:.0f} cm")
print(f"Luas = {alas*tinggi/2:.0f}")